controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -55,7 +56,6 @@ import (
 
 var (
 	nodeName                                string
-	nodeConfigurationTimeout                = 5 * time.Minute
```

Comment:
We don't need this ?

Reply:
Since we don't need to check how long a node is applying the policy, I think we can remove it, it's not being used except for setting it

Reply:
You mean that we already have a timeout mechanism when we run `nmstatectl set` so we don't need any of this right ?

Reply:
Yes, I believe it's enough to rely on the timeout of `ApplyDesiredState` and only check if there is a node with Failed enactment